src/engine/source/kvdb2/src/kvdbManager.cpp

```diff
@@ -67,100 +89,209 @@ void KVDBManager::initializeOptions()
 
 void KVDBManager::initializeMainDB()
 {
-    std::vector<std::string> columnNames;
+    auto dbStoragePath = m_ManagerOptions.dbStoragePath.string();
 
-    auto dbPath = fmt::format("{}{}", m_ManagerOptions.dbStoragePath.string(), m_ManagerOptions.dbName);
+    std::filesystem::create_directories(dbStoragePath);
 
-    std::filesystem::create_directories(dbPath);
+    auto dbNameFullPath = fmt::format("{}{}", dbStoragePath, m_ManagerOptions.dbName);
+
+    std::vector<std::string> columnNames;
+    auto listStatus = rocksdb::DB::ListColumnFamilies(rocksdb::DBOptions(), dbNameFullPath, &columnNames);
 
-    auto listStatus = rocksdb::DB::ListColumnFamilies(rocksdb::DBOptions(), dbPath, &columnNames);
+    std::vector<rocksdb::ColumnFamilyDescriptor> cfDescriptors;
+    std::vector<rocksdb::ColumnFamilyHandle*> cfHandles;
 
     if (listStatus.ok())
     {
         for (auto cfName : columnNames)
         {
             auto newDescriptor = rocksdb::ColumnFamilyDescriptor(cfName, rocksdb::ColumnFamilyOptions());
-            m_cfDescriptors.push_back(newDescriptor);
+            cfDescriptors.push_back(newDescriptor);
         }
     }
     else
     {
-        auto newDescriptor = rocksdb::ColumnFamilyDescriptor(DEFAULT_CF_NAME, rocksdb::ColumnFamilyOptions());
-        m_cfDescriptors.push_back(newDescriptor);
+        auto newDescriptor =
+            rocksdb::ColumnFamilyDescriptor(rocksdb::kDefaultColumnFamilyName, rocksdb::ColumnFamilyOptions());
+        cfDescriptors.push_back(newDescriptor);
     }
 
-    auto openStatus = rocksdb::DB::Open(m_rocksDBOptions, dbPath, m_cfDescriptors, &m_cfHandles, &m_pRocksDB);
+    auto openStatus = rocksdb::DB::Open(m_rocksDBOptions, dbNameFullPath, cfDescriptors, &cfHandles, &m_pRocksDB);
 
-    for (int k=0; k<m_cfDescriptors.size(); k++)
+    for (int k = 0; k < cfDescriptors.size(); k++)
     {
-        m_mapCFHandles.insert(std::make_pair(m_cfDescriptors[k].name, m_cfHandles[k]));
+        m_mapCFHandles.insert(std::make_pair(cfDescriptors[k].name, cfHandles[k]));
     }
```

Comment:
Could we please use more descriptive names for variables instead of `i`, `j` or `k`?